<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zetaChat</class>
 <widget class="QWidget" name="zetaChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="3,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,1">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Чат</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QListWidget" name="lwLog"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Пользователи</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QListWidget" name="lwUsers">
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1">
       <item>
        <widget class="QPlainTextEdit" name="pteMessage"/>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,3,1">
         <item row="0" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Выход</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pbSend">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Отправить</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="cbToAll">
           <property name="text">
            <string>Всем</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
